tools/downloader/topologies_test.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+import os
+import numpy as np
+import cv2 as cv
+
+import topologies as omz
+
+from pathlib import Path
+from parameterized import parameterized
+from openvino import inference_engine
+from openvino.inference_engine import IECore
+
+class TestTopologies(unittest.TestCase):
+    def test_load_intel(self):
+        face_detection = omz.Model('face-detection-0100')
+        xml_path, bin_path = face_detection.get_ir(precision='FP32')
+        
+        self.assertTrue(os.path.exists(xml_path))
+        self.assertTrue(os.path.exists(bin_path))
+
+        ie = IECore()
+        net = ie.read_network(xml_path, bin_path)
+        exec_net = ie.load_network(net, 'CPU')
+        infer_request = exec_net.requests[0]
+
+        input_blob = infer_request.input_blobs[next(iter(net.input_info))]
+        output_blob = infer_request.output_blobs[next(iter(net.outputs))]
+
+        self.assertEqual(input_blob.tensor_desc.dims, [1, 3, 256, 256])
+        self.assertEqual(output_blob.tensor_desc.dims, [1, 1, 200, 7])
+
+    @parameterized.expand([[None, 'FP16-INT8'], ['FP16-INT8', None],
+                           ['FP16-INT8', 'FP16-INT8']])
+    def test_load_intel_INT8(self, init_precision, get_ir_precision):
```

Comment:
It seems like this test case is a superset of the one above - it uses a different model and a different precision, but otherwise it's equivalent to the `[None, 'FP16-INT8']` instance of this test case.